test/server/cards/01_SOR/WingLeader.spec.js

```diff
@@ -0,0 +1,34 @@
+describe('Wing Leader', function() {
+    integration(function() {
+        describe('Wing Leader\'s ability', function() {
+            beforeEach(function () {
+                this.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['wing-leader'],
+                        groundArena: ['21b-surgical-droid', 'fleet-lieutenant', { card: 'rebel-pathfinder', upgrades: ['experience'] }],
+                    },
+                    player2: {
+                        groundArena: ['viper-probe-droid']
+                    }
+                });
+            });
+
+            it('can give two experience to a unit', function () {
+                this.player1.clickCard(this.wingLeader);
+                expect(this.player1).toBeAbleToSelectExactly([this.fleetLieutenant, this.rebelPathfinder]);
+                this.player1.clickCard(this.fleetLieutenant);
+
+                expect(this.fleetLieutenant.upgrades.map((upgrade) => upgrade.internalName)).toEqual(['experience', 'experience']);
```

Comment:
I actually just merged in a new test method that makes this a little easier. Do you mind updating this to be:

`expect(this.fleetLieutenant).toHaveExactUpgradeNames(['experience', 'experience']);`

I already updated the TIE Advanced test, so this way everything will be consistent and other people will know to use this method in the future if they look at this code as an example. Ditto for the expectation on line 30.

Reply:
will do.